webpack/assets/javascripts/react_app/components/BreadcrumbBar/components/__snapshots__/Breadcrumb.test.js.snap

```diff
@@ -1,31 +1,29 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`Breadcrumbs renders breadcrumbs menu 1`] = `
-<Breadcrumb
-  title={false}
->
+<Component>
```

Comment:
is this expected?

Reply:
Based on https://enzymejs.github.io/enzyme/docs/api/ReactWrapper/name.html

> The order of precedence on returning the name is: `type.displayName` -> `type.name` -> `type`.

I assume that PF4 compiled it into an anonymous function and didn't set a `displayName` or `name`.

Reply:
Should we set the `displayName` manually?

Reply:
I don’t think we should as it's a 3rd party component, as far as we keep our snapshots short, we should be fine.

Reply:
PF4 fixed the issue with the display name: https://github.com/patternfly/patternfly-react/pull/4391

This PR in foreman-js should fix that issue: https://github.com/theforeman/foreman-js/pull/146